RevenueCatUI/CustomerCenter/Data/PurchaseInformation.swift

```diff
@@ -32,9 +32,7 @@ struct PurchaseInformation {
          explanation: Explanation,
          price: PriceDetails,
          expirationOrRenewal: ExpirationOrRenewal?,
-         willRenew: Bool,
          productIdentifier: String,
-         active: Bool,
```

Comment:
These were only used in this file and we don't need them anymore since the information is in the product